src/components/searchBar/SearchBar.jsx

```diff
@@ -4,31 +4,30 @@ import ResultCard from '../ResultCard/ResultCard';
 import './Search.css';
 
 const Search = () => {
-  const [querys, setQuerys] = useState('');
+  const [query, setQuery] = useState(['']);
```

Comment:
Nope!
If `query` is singular (which seems legit) it should *never* contain an Array

Reply:
Oops i havent understood your comment like this